.gitignore

```diff
@@ -18,6 +18,7 @@ development_app
 
 # rubocop cached remote config
 .rubocop-https---raw-githubusercontent-com-decidim-decidim-master--rubocop-yml
+.rubocop-https---raw-githubusercontent-com-decidim-decidim-master--rubocop-rails-yml
```

Comment:
Mmm, maybe a .rubocop-https-* rule would be simpler?

Reply:
This tracks the exact files that should be ignored. If you have 3 rubocop caching files for some reason, that could affect rubocop behavior so it'd be better to let the user know one of them is unexpected...

Reply:
:+1:

Reply:
That said, that's really unlikely to happen or even be a problem, so this comes down to style. I prefer explicitness but I understand it leads to complicated rules, so totally fine with changing it :+1:

Reply:
Hehe, I didn't see your thumbs up before posting my comment. I'll leave it as it is if I convinced you :)

Reply:
Yes, I think the benefit you commented is bigger than the drawback of some extra lines in the `.rubocop` file